requirements/requirements-gridsync.txt

```diff
@@ -1,6 +1,9 @@
 asn1crypto==0.24.0 \
     --hash=sha256:2f1adbb7546ed199e3c90ef23ec95c5cf3585bac7d11fb7eb562a3fe89c64e87 \
     --hash=sha256:9d5c20441baf0cb60a4ac34cc447c6c189024b6b4c6cd7877034f4965c464e49
+atomicwrites==1.3.0 \
```

Comment:
fwiw, this is also what `twisted.python.filepath.FilePath.setContent` is for.

Reply:
I was not aware of `twisted.python.filepath.FilePath.setContent` -- thanks! It looks like Windows support is [lacking for `setContent`](https://twistedmatrix.com/documents/current/api/twisted.python.filepath.FilePath.html#setContent), however, in comparison to [atomicwrites](https://pypi.org/project/atomicwrites/).